packages/hash/frontend/src/blocks/page/createSuggester/createSuggester.tsx

```diff
@@ -179,10 +204,20 @@ export const createSuggester = (
           docChangedInTransaction(tr)
             ? null
             : tr.mapping.map(state.suggestedBlockPosition);
-        const trigger = findTrigger(nextEditorState);
+        const trigger =
+          state.trigger?.triggeredBy === "event" &&
+          findTrigger(nextEditorState) === null
+            ? state.trigger
+            : findTrigger(nextEditorState);
```

Comment:
You probably don't want to run `findTrigger` twice like this

Reply:
Done in e9ddc7c6a98b418640099449c507e7268f4a8e9c.

Reply:
Much clearer, thanks Luís!